### ISSUE TITLE
my first evil package!

### ISSUE THREAD

**Issue by ninrod** [d12frosted/elpa-mirror#4]
hey @d12frosted, with I'm justing letting you know, as you are a core spacemacs dev, that I've just pushed my first package to melpa, which adds an evil text object for HTML/XML tag attributes. 

check it out when you have some time: https://github.com/ninrod/exato. Maybe add to the evil layer if you find it useful. 

Cheers!

**Comment by d12frosted** [comment]
Hey @ninrod,

That's great. Kudos! It's [already in my mirror](https://github.com/d12frosted/elpa-mirror/blob/59bcf4276502c3d6af4fad6f5f15f4074e51b491/melpa/exato-20171119.1149.el) . 😸 

Unfortunately, I am not as active as I used to be, so in case you wish to add it to Spacemacs, you'd better open up an issue in main Spacemacs repo. 

P. S. But I hope to return my activity very soon 😈 

P. P. S. Closing this issue as slightly irrelevant to this repo.
